### ISSUE TITLE
Sending multi bundle assets using HW Wallet does not work

### ISSUE THREAD

**Issue by DevStakePool** [input-output-hk/lace#108]
Hi,

With an account connected with Ledger Nano, I tried to send a few ADA to 15 different addresses, using the UI to construct a multi bundle TX.

The operation went well but the transaction was never submitted to the blockchain. The Ledger device signed the TX correctly, showing all the 15 output addresses with the respective ADA value, but once the TX was signed it was never published on-chain and the UI did not report anything

**Comment by Fka27ADA-mod** [comment]
@DevStakePool Talk to Cardano Support on the [Live support page](https://explore-cadarno.org/) so that your issue can be looked into.

**Comment by Scottyfis** [comment]
@DevStakePool have you solved your issue?

**Comment by DevStakePool** [comment]
> @DevStakePool Talk to Cardano Support on the [Live support page](https://explore-cadarno.org/) so that your issue can be looked into.

Is this scam?

**Comment by Scottyfis** [comment]
no it's not scam, why do you ask?

**Comment by DevStakePool** [comment]
![image](https://github.com/input-output-hk/lace/assets/96963922/dce6a565-c2d4-424f-9edf-c2dad7a51072)
You tell me!

**Comment by Scottyfis** [comment]
oh it's the site click visit this unsafe site it's going to display maybe some external Ip is installed on your site.

**Comment by michalrus** [comment]
> Is this scam?

Yes, cyber-squatting, look closely at the domain name – containing ca**dar**no instead of ca**rda**no

**Comment by rhyslbw** [comment]
Sorry @DevStakePool, I didn't mean to mark your comment as _abuse_, only that it contained a URL that users could still access

**Comment by vix-io** [comment]
Unable to reproduce; I successfully send out a tx from a Nano X to 10 recipients -> https://cardanoscan.io/transaction/f477211cde92b419bfb93daf6c38b4dedcb4a54e604a899801492e91070fb821
